### PR TITLE
Hls examples new clean

### PR DESCRIPTION
**Pull request by bmesnet**
HLS search : align writes to DDR on 64 bytes to get data read ok
HLS checksum : push code that is used for benchmark -- requires SDRAM_USED=FALSE since ddr port removed

### CODE REVIEW

File: hardware/action_examples/hls_search/hls_search/.apc/autopilot.apfmapping [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<autopilotfilemapping:AutoPilotFileMapping xmlns:autopilotfilemapping="www.autoesl.com/autopilotfilemapping">
+  <source>
+    <originFiles name="hls_search.cpp" path="/afs/vlsilab.boeblingen.ibm.com/proj/fpga/framework/mesnet/snap/hardware/action_examples/hls_search/hls_search.cpp"/>
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/.apc/autopilot.apfmapping]: what do you need this for? Remove!!! it contains ibm specific stuff.

File: hardware/action_examples/hls_search/hls_search.cpp [hunk 2]
```diff
@@ -745,7 +751,7 @@ adding_padding_to_256bytes_to ensure test  ok
      */
 
     // read data for text
-    fp = fopen("../../../../snap_search123.txt", "r");
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search.cpp]: brrrr. so many .. ... .. . I do not like this, can we make this simpler or differently? Is that the test bench code?

File: hardware/action_examples/hls_search/hls_search/.project [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/.project]: Do we need this? Did you do a git commit -a without reviewing the files you were adding. Please remove if not needed.

File: hardware/action_examples/hls_search/hls_search/.settings/hls_search.Release.launch [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/.settings/hls_search.Release.launch]: Some here. ...

File: hardware/action_examples/hls_search/hls_search/.vivado_hls_log_all.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/.vivado_hls_log_all.xml]: And here.

File: hardware/action_examples/hls_search/hls_search/.vivado_hls_log_all.xml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+<vivadoHLSLog:LogRoot xmlns:vivadoHLSLog="www.xilinx.com/vivadoHLSLog">
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/.vivado_hls_log_all.xml]: And here.

File: hardware/action_examples/hls_search/hls_search/solution1/.autopilot/db/autopilot.flow.log [hunk 7]
```diff
@@ -0,0 +1,171 @@
+INFO: [HLS 200-0] Workspace /afs/vlsilab.boeblingen.ibm.com/proj/fpga/framework/mesnet/snap/hardware/action_examples/hls_search/hls_search/solution1 opened at Thu Jun 01 15:45:14 CEST 2017
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/solution1/.autopilot/db/autopilot.flow.log]: Brrr.

File: hardware/action_examples/hls_search/hls_search/solution1/directives.tcl [hunk 8]
```diff
@@ -0,0 +1,5 @@
+############################################################
```
Comment (fhaverkamp) [inline hardware/action_examples/hls_search/hls_search/solution1/directives.tcl]: Are you sure you like to add this??

**Comment by fhaverkamp** [comment]
Please remove obsolete stuff and code with Xilinx copyright. Thanks.

**Comment by bmesnet** [comment]
seems that my merge with master created some strange things !! recreating a new clean branch

File: hardware/action_examples/hls_search/hls_search.cpp (hunk 2 above)
Comment (bmesnet) [inline hardware/action_examples/hls_search/hls_search.cpp]: yes that's the testbench

**Pull request closed without merging**